test/plugin/scenarios/webflux-scenario/webflux-projectB-scenario/src/main/java/test/apache/skywalking/apm/testcase/sc/webflux/projectB/controller/DbOperate.java

```diff
@@ -0,0 +1,15 @@
+package test.apache.skywalking.apm.testcase.sc.webflux.projectB.controller;
+
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+
+@Service
+public class DbOperate {
+    // This method is to generate an annotation trace for testing
```

Comment:
What does this method? Why doesn't a DBOperate include a real database access?

Reply:
In fact, after being processed by the Schedulers.elastic() method, all traces will be split into a single piece, so an Annotiation trace can be used here to simulate the subsequent span split. Later I will modify to h2 based DB access.

Reply:
![image](https://user-images.githubusercontent.com/22290226/158324682-67e696d0-9ce3-4a26-8e37-a5cfae1a7ee7.png)